vault/resource_kubernetes_secret_backend.go

```diff
@@ -0,0 +1,179 @@
+package vault
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/util"
+)
+
+const (
+	fieldKubernetesHost    = "kubernetes_host"
+	fieldKubernetesCACert  = "kubernetes_ca_cert"
+	fieldServiceAccountJWT = "service_account_jwt"
+	fieldDisableLocalCAJWT = "disable_local_ca_jwt"
+)
+
+func kubernetesSecretBackendResource() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: kubernetesSecretBackendCreateUpdate,
+		ReadContext:   kubernetesSecretBackendRead,
+		UpdateContext: kubernetesSecretBackendCreateUpdate,
+		DeleteContext: kubernetesSecretBackendDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: getKubernetesSecretsMountSchema(),
+	}
+}
+
+func getKubernetesSecretsMountSchema() map[string]*schema.Schema {
+	s := getMountSchema("type")
+	for k, v := range getKubernetesSecretsSchema() {
```

Comment:
Suggest replacing with `mustAddSchema()`  https://github.com/hashicorp/terraform-provider-vault/blob/b914b1333431cda7c29180818359411c16145c0e/vault/provider.go#L810

Reply:
Nice, didn't know about that function. Done in 57179fd.